nix/lib/mkBunNodeModules.nix

```
@@ -8,6 +8,10 @@
   ...
 }:
```

```suggestion
{
  packages,
  dontPatchShebangs ? false,
  ...
}:
```

For explicitness `mkBunNodeModules` should probably just take a set directly now that it has an option associated with it